scripts/write-credentials.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This file writes out the google credentials required to access the google drive api to read our team images.
```

Comment:
This whole thing is a little annoying. So far as I'm aware you _can't_ feed these credentials into google's node sdk. They literally have to be in a file. I'm assuming that's just the standard way they do things, but it's annoying all the same. 

Thankfully, there's a hack for that. I'm saying that enough lately that I should print stickers and tshirts.